content/zh/docs/devops-user-guide/examples/a-maven-project.md

```diff
@@ -12,9 +12,9 @@ weight: 11430
 - 您需要有一个 [Docker Hub](http://www.dockerhub.com/) 帐户。
 - 您需要创建一个企业空间、一个 DevOps 项目和一个用户，并需要邀请该用户至 DevOps 项目中并赋予 `operator` 角色。有关更多信息，请参见[创建企业空间、项目、用户和角色](../../../quick-start/create-workspace-and-project/)。
```

Comment:
The link of `create-workspace-and-project` seems linked to a '404' error page, it should be '../../../quick-start/create-workspace-and-project.md'

Reply:
Hi, thanks for your review. The link here is correct.

Reply:
Sorry for the trouble.

Reply:
No worries.